index.html

```diff
@@ -170,15 +202,18 @@ <h4 class="set-h">Skills</h4>
             </ul>
             <img class="set-img" src="images/ic_skills.png">
         </div>
+
+        <img class="dotImg1" src="images/image_dots_with_teal.svg">
         </section>
         <br>
     
 
         <section class="contact">
             <div class="corner__geometry">
-                <img src="./images/corner_geometry.svg">
+                <img class="corner_geometry" src="./images/corner_geometry.svg">
+                <img class="image-geometry_6" src="./images/image-geometry_6.svg">
             </div>
-
+        <div class="get_started">
             <h2 class="h2">Get Started</h2>
```

Comment:
- [ ] Kindly increase the `font-size` of this header to match the one in Figma 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/200125103-1147b3a7-a3c6-46c4-aa61-5266dac903d0.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/200125111-fa17a990-7820-4bd9-882f-412a06f1cd70.png)

Reply:
Fixed.